travis/deploy_pull_request.sh

```diff
@@ -26,7 +26,19 @@ function publish_pull_request_urls_in_github {
   # Done so because every PR is an issue, and the issues api allows to post general comments,
   # while the PR api requires that comments are made to specific files and specific commits
   GITHUB_PR_COMMENTS=https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments
-  curl -H "Authorization: token ${GITHUB_GNOSIS_INFO_API_TOKEN}" --request POST ${GITHUB_PR_COMMENTS} --data '{"body":"Travis automatic deployment:\r\n '${REVIEW_FEATURE_URL}'"}'
+  PREDICATE='gnosis-info'
+
+  # Check GITHUB_PR_COMMENTS if `gnosis-info` exists
+  # If present, do nothing as we want to reduce noise
+  # Else, comment URL
+  IS_COMMENT_PRESENT="$(node ./travis/check_pr_comments.js ${GITHUB_PR_COMMENTS} ${PREDICATE})"
```

Comment:
While not an exact equivalent, you can use
```
IS_COMMENT_PRESENT=$(curl https://api.github.com/repos/gnosis/dex-react/issues/129/comments | grep -q '"login":\s*"gnosis-info"' && echo "true" || echo "false")
```
And skip JS and extra dependencies

Reply:
I stick to what i know i guess haha

Reply:
resolved in following commits